### DISCUSSION TITLE
0.0.62

### DISCUSSION THREAD

**Discussion by Xzandro** [Xzandro/sw-exporter#515] [General]
* Added Get & Install Cert (Steam) button, thanks @nf-11 
* Adjusted help section for the Steam mode

<hr /><em>This discussion was created from the release <a href='https://github.com/Xzandro/sw-exporter/releases/tag/v0.0.62'>0.0.62</a>.</em>

**Comment by QiFailed** [comment]
When i will start Proxy i become Java Script error

**Comment by Facafatatronche** [comment]
Hi!
I try to download sw exporter to get the json file hpwever I get this JavaScript error. I cannot find anything online (that I can understand) regarding this issue... I try to disable SWARFARM as recommended in another post but it doesn't change anything. I also download the latest version of Java but still nothing. Can anyone help me please?

Thanks a lot!
![image](https://github.com/Xzandro/sw-exporter/assets/173823182/8a414479-4f89-4a6a-83f9-17196cbb5f1e)
